app/scripts/metamask-controller.js

```diff
@@ -2019,22 +2019,22 @@ export default class MetamaskController extends EventEmitter {
         'AccountsController:listAccounts',
         'AccountsController:getSelectedAccount',
         'AccountsController:setSelectedAccount',
+        'NetworkController:getState',
+        'NetworkController:setActiveNetwork',
       ],
     });
 
     this.mmiController = new MMIController({
-      messenger: mmiControllerMessenger,
+      messagingSystem: mmiControllerMessenger,
```

Comment:
Related to a comment I made above, should this be `messenger` for consistency with controllers?

``` suggestion
      messenger: mmiControllerMessenger,

Reply:
It's a bit confusing — BaseControllerV2 takes a `messenger` option and stores this under a private `messagingSystem` property. Therefore, a consumer uses `messenger` to pass the messenger to the constructor but the controller uses `this.messagingSystem` internally to use the messenger. You can verify this by looking at controllers in `core` that inherit from BaseController v2.